### DISCUSSION TITLE
Niveau sonore des MP3

### DISCUSSION THREAD

**Discussion by benjapan** [marian-m12l/studio#303] [General]
Tout d'abord, mille mercis pour ce superbe travail d'interfaçage avec la Lunii.
J'y ai mis une playlist des Beatles pour mes filles :)

A ce propos, les nouveaux fichiers audio MP3 ajoutés semblent d'un niveau sonore bien moins élevé que les fichiers audio d'origine.

Est-ce un problème connu ? Hormis le fait de devoir booster le gain et potentiellement saturer les fichiers, y'a-t-il une issue ouverte sur ce sujet ?

**Comment by fparrel** [comment]
J'ai mis les commandes que j'utilise à la fin du wiki https://github.com/marian-m12l/studio/wiki/Documentation
Ca permet de ne pas avoir de saturation tout en boostant le volume

**Comment by Noday31** [comment]
Bonjour,
moi aussi j'ai un sacré soucis au niveau du niveau sonore (inutilisable en voiture même avec le casque) j'ai bien vu ca:
# Record sound via command line, press Ctrl+C to stop
arecord --rate 44100 -vv -fdat /tmp/out.wav
# Compute needed gain
A=$(echo "-0.1-( $(ffmpeg -i /tmp/out.wav -af 'volumedetect' -f null /dev/null 2>&1|grep max_volume|cut -f 5 -d" ") )"|bc -l)
# Convert to mp3 and ajust volume
ffmpeg -i /tmp/out.wav -vn -af "volume=${A}dB" /tmp/out.mp3
# Move file to output and clean
mv -f /tmp/out.mp3 $1
rm /tmp/out.wav

Mais bon... je ne sais pas ou le mettre et/ou ou le modifier .. je suis sous Windows 
@fparrel tu peux m'aiguiller?

Merci
